docs/modules/operation/pages/admin/webui/log-reader.adoc

```diff
@@ -0,0 +1,17 @@
+[[log-statistics]]
+= Instrumentation Log Statistics
+
+The instrumentation log reader determines how long data collection takes for each node in the system, and which nodes are taking the most time.
+You can view this information in the Instrumentation Log Statistics UI (menu:Gears icon[Additional Tools>Instrumentation Log Reader].
+
+The instrumentation log reader requires instrumentation logs produced with a DEBUG level.
```

Comment:
The instrumentation log is defaulted to INFO, which is enough to provide data.

Reply:
Thanks. The wiki content was more of a spec. Must have changed in the actual development.